presto-native-execution/presto_cpp/main/PrestoServer.cpp

```diff
@@ -160,6 +160,8 @@ void registerVeloxCudf() {
 #ifdef PRESTO_ENABLE_CUDF
   // Disable by default.
   velox::cudf_velox::CudfConfig::getInstance().enabled = false;
+  facebook::velox::cudf_velox::CudfOptions::getInstance().setPrefix(
```

Comment:
This has now changed. CI does not test this.
`velox::cudf_velox::CudfConfig::getInstance().functionNamePrefix = SystemConfig::instance()->prestoDefaultNamespacePrefix();`